src/org/exist/backup/Main.java

```diff
@@ -459,9 +402,20 @@ private static void shutdown(Collection root) {
     }
 
 
-    public static void main(String[] args) {
+    public static void main(final String[] args) {
         try {
-            process(args);
+            final ParsedArguments arguments = CommandLineParser
+                    .withArguments(userArg, passwordArg, dbaPasswordArg)
+                    .andArguments(backupCollectionArg, backupOutputDirArg)
+                    .andArguments(restoreArg, rebuildExpathRepoArg)
+                    .andArguments(helpArg, guiArg, quietArg, optionArg)
+                    .parse(args);
+
+            process(arguments);
+        } catch(final ArgumentException e) {
+            System.out.println(e.getMessageAndUsage());
+            System.exit(2);
```

Comment:
`128` is reserved for when you give an invalid number (i.e. `0 > n > 255`) to `exit`